### PR TITLE
Json http path exists

### PR DESCRIPTION
**Pull request by tcnh**
Added a method to json http test to check if json path exists in a json file.
Note: when the key exists, but value is explicitly set to null, this will return false

This PR seems huge because I branched from my fork that was pretty far behind. Only the four relevant files are changed though.

### CODE REVIEW

**Comment by fhoeben** [comment]
Was your fork based on the updated repo version (with the smaller size #130)? I believe it is not.
I'm afraid merging this will bring back all history (and the large size). Can you create a patch containing this PR's changes and apply that on a 'clean fork'?

File: src/main/java/nl/hsac/fitnesse/fixture/util/JsonPathHelper.java [hunk 1]
```diff
@@ -68,6 +68,18 @@ public String updateJsonPathWithValue(String json, String jsonPath, Object value
         }
     }
 
+    public boolean jsonPathExists(String json, String jsonPath) {
+        boolean result = false;
+        try {
+            result = null != getJsonPath(json, jsonPath);
+        } catch (RuntimeException e) {
+            if(e.getMessage().contains("multiple")) {
```
Comment (fhoeben) [inline src/main/java/nl/hsac/fitnesse/fixture/util/JsonPathHelper.java]: space after `if`, before `(`

File: wiki/FitNesseRoot/HsacAcceptanceTests/SlimTests/HttpTest/JsonHttpTest.wiki [hunk 2]
```diff
@@ -31,7 +31,9 @@
 |set json path|airplanes[0].seats   |to                  |300                                                 |
 |check        |json path            |airplanes[0].seats  |300                                                 |
 |show         |response                                                                                       |
-
+|ensure       |json path exists     |airplanes[0].seats                                                       |
+|reject       |json path exists     |airplanes[0].beds                                                        |
+|ensure       |json path exists     |airplanes[*].airline                                                     |
```
Comment (fhoeben) [inline wiki/FitNesseRoot/HsacAcceptanceTests/SlimTests/HttpTest/JsonHttpTest.wiki]: Can you add a line
```
|reject       |json path exists     |airplanes[*].beds |
```

**Comment by tcnh** [comment]
Creating new PR based on clean fork. Aso implementing suggested changes.
Nice catch with the added test btw. that situation was unhandled :)

**Pull request closed without merging**